pages/deployment/[uuid].tsx

```diff
@@ -193,7 +194,9 @@ export default function Deployment() {
         <DialogContent>
           <DialogContentText sx={{ backgroundColor: 'whitesmoke', p: 2 }}>
             <pre>
-              <span># Login to Docker</span>
+              <span>
+                # Login to Docker (your token can be found on the <Link href='/settings'>/settings</Link> page)
+              </span>
               <br />
```

Comment:
Can we use MUI Boxes here and add styling instead of using breaks? Can't remember if we can use MUI components inside the <pre> tags or not!